Swift/Database.swift

```diff
@@ -220,6 +220,27 @@ public final class Database {
     }
     
     
+    /// Sets an expiration date on a document.
+    /// After this time the document will be purged from the database.
+    ///
+    /// - Parameters:
+    ///     - id: The ID of the document to set the expiration date for
+    ///     - date: The expiration date. Set nil date will reset the document expiration.
+    /// - Throws: An error on a failure.
+    public func setDocumentExpiration(id: String, date: Date?) throws {
+        try _impl.setDocumentExpirationWithID(id, date: date)
+    }
+    
+    
+    /// Returns the expiration time of a document, if exists, else nil.
+    ///
+    /// - Parameter id: The ID of the document to set the expiration date for.
+    /// - Returns: the expiration time of a document, if one has been set, else nil.
+    public func getDocumentExpiration(id: String) -> Date? {
```

Comment:
Let's do `getDocumentExpiration(withID documentID: String) -> Date?` for consistency.